sdk/keyvault/Azure.ResourceManager.KeyVault/tests/SessionRecords/VaultOperationsTests/KeyVaultManagementListDeletedVaults.json

```diff
@@ -5088,6 +5088,7 @@
       "RequestMethod": "POST",
       "RequestHeaders": {
         "Authorization": "Sanitized",
+        "Content-Length": "0",
```

Comment:
Will this be set to 0 automatically when re-recording? I would prefer not to have to remember to do this every time I re-record (fairly often in data-plane projects).

Reply:
Yes it will be set automatically when re-recording.